proto/ibc/applications/interchain_accounts/v1/types.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package ibc.applications.interchain_accounts.v1;
+
+import "google/protobuf/any.proto";
+import "gogoproto/gogo.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/27-interchain-accounts/types";
+
+// Raw tx body
+message IBCTxRaw {
+  bytes body_bytes = 1 [(gogoproto.moretags) = "yaml:\"body_bytes\""];
+}
+
+// Body of a tx for an ics27 IBC packet
+message IBCTxBody {
+  repeated google.protobuf.Any messages = 1;
+}
+
+// The different types of interchain account transactions
+// EXECUTE_TX is used when sending a TX from the controller side to the host side. The host side will execute the tx on
+// behalf of the interchain account.
+enum Type {
+  option (gogoproto.goproto_enum_prefix) = false;
+  // Execute message type
+  TYPE_EXECUTE_TX_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "EXECUTE_TX"];
```

Comment:
Will there ever be more than just execute tx?

Reply:
Potentially no, but I think it's worth leaving in for extensibility.

Reply:
Yea I just figure it isn't backwards compatible anyways. If someone uses an enum with value 1, the current implementation couldn't do anything about it and would return an error

Reply:
Yeah, it's being passed in the packet. The packet is made up of type + data bytes. I'll think about it though, good points.